pyesbulk/__init__.py

```diff
@@ -8,27 +8,72 @@
 import logging
 import math
 import time
+import inspect
+import importlib
+from pathlib import Path
 
 from collections import Counter, deque
 from datetime import datetime, tzinfo, timedelta
 from random import SystemRandom
 
-try:
-    from elasticsearch1 import (
-        VERSION as es_VERSION, helpers, exceptions as es_excs
-    )
-    _es_logger = "elasticsearch1"
-except ImportError:
-    from elasticsearch import (
-        VERSION as es_VERSION, helpers, exceptions as es_excs
+# We're going to dynamically bind imported modules to these names that
+# match with the Elasticsearch object passed by our client. These
+# declarations serve mostly as documentation, since they're not necessary
+# for Python, and don't appear to help flake8 analysis. Note that uses of
+# these names in code are annotated with "noqa" line comments to avoid
+# failing the flake8 scan.
+global helpers
```

Comment:
You might want to just use:

```
helpers = None
es_excs = None
```

You only need the `global` in places where you want to be sure the reference is forced globally.

Reply:
Good point; and that quiets `flake8` too, without the `noqa` comments.